docs/src/inputs/button/button.jsx

```diff
@@ -1,194 +1,191 @@
-
-                {/* Inverse */}
-                <Header anchor="color-inverted" size="large" style={{ marginTop: '55px' }} sub>
-                    Inverse
-                    <Header.Subheader>
-                        Button colors can be inverted.
-                    </Header.Subheader>
-                </Header>
-
-                <ButtonInverseExample />
-
-                <Highlighter customStyle={{ marginBottom: '44px', marginTop: '44px' }}>
-                    {require('!!raw-loader!./buttonInverseExample').default}
-                </Highlighter>
-
-                {/* Outlined */}
-                <Header anchor="color" size="large" style={{ marginTop: '55px' }} sub>
-                Outlined
-                    <Header.Subheader>
-                        Buttons can have different border colors.
-                    </Header.Subheader>
-                </Header>
-
-                <ButtonOutlineExample />
-
-                <Highlighter customStyle={{ marginBottom: '44px', marginTop: '44px' }}>
-                    {require('!!raw-loader!./buttonOutlineExample').default}
-                </Highlighter>
-
-                {/* Relax */}
-                <Header anchor="relax" size="large" style={{ marginTop: '55px' }} sub>
-                    Relax
-                    <Header.Subheader>
-                        A button can have relaxed padding.
-                    </Header.Subheader>
-                </Header>
-
-                <ButtonRelaxExample />
-
-                <Highlighter customStyle={{ marginBottom: '44px', marginTop: '44px' }}>
-                    {require('!!raw-loader!./buttonRelaxExample').default}
-                </Highlighter>
-
-                {/* Width */}
-                <Header anchor="width" size="large" style={{ marginTop: '55px' }} sub>
-                    Width
-                    <Header.Subheader>
-                        A button can have a fixed width.
-                    </Header.Subheader>
-                </Header>
-
-                <ButtonWidthExample />
-
-                <Highlighter customStyle={{ marginBottom: '44px', marginTop: '44px' }}>
-                    {require('!!raw-loader!./buttonWidthExample').default}
-                </Highlighter>
+                <MarkdownContainer>
+                    <Typography
+                        className="description"
+                        variant="body1"
+                    >
+                        {description}
+                    </Typography>
+
+                    {/**
+                     * Solid Button
+                     */}
+                    <Heading
+                        anchorLink="solid-button"
+                        variant="h2"
+                    >
+                        Solid Button
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Types
+                    </Typography>
```

Comment:
Types...?  😕 
Types of what?
Is this perhaps a copy-and-paste error, and this is meant to say something else?

Reply:
![image](https://user-images.githubusercontent.com/1100790/131591257-2980b709-9bb7-41a1-8ec0-2ba6cf9e5dff.png)

Reply:
Makes sense.  Thanks!  Resolving this.